docs/samplesheets/examples.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Channel usage examples
+description: Examples of advanced sample sheet creation techniques.
+---
+
+# Sample sheet channel manipulation examples
+
+## Separate entries based on a condition
+
+You can use the [`.branch()` operator](https://www.nextflow.io/docs/latest/operator.html#branch) to separate the channel entries based on a condition. This is especially useful when you can get multiple types of input data.
+
+This example shows a channel which can have entries for WES or WGS data. These analysis are different so we want to separate the WES and WGS entries from eachother. We also don't want the `bed` file input for the WGS data, so the resulting channel with WGS data should not have this file in it.
+
+```groovy
+// example channel:
+// [[id:example, type:WGS], WGS.bam, WGS.bam.bai, []]
+// [[id:example2, type:WES], WES.bam, WES.bam.bai, WES.bed]
```

Comment:
we are showing the channel view below anyway....rather have the samplesheet here along with header?

Reply:
This is the whole point, the first view shows the channel before the `.branch()` manipulation. The last view shows the channel after the manipulation

Reply:
sure I was just thinking a view of the sample sheet also helps with easily guiding a beginner through the flow

Reply:
True, but that's already explained in other parts of the documentation? I think it's advised to read the `.fromSamplesheet()` docs before reading the examples (@ewels and @mirpedrol what do you think of this?)

Reply:
Agree with @nvnieuwk, I missed that both channels were different. Maybe we can change "example channel" by "channel created by fromSamplesheet() previous to modification" instead?

Reply:
cool

Reply:
I haven't touched these docs much and yeah, had to read this several times to understand the point.

I've rewritten to use descriptions of the channels instead of examples:

```diff
- // [[id:example, type:WGS], WGS.bam, WGS.bam.bai, []]
+ // Channel has 3 elements: meta, bam, bai
```